nginx-sidecar/helm/nginx-sidecar/values.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  repository: gcr.io/ggl-cas-storage/cas-nginx
+  tag: 0.1.4 # Matches the chart version since the image gets tagged automatically from that
```

Comment:
can you add a comment explaining why we need this and can't use `.Chart.version` ?